helmfile.d/upstream/aquasecurity/trivy-operator/templates/rbac/clusterrole.yaml

```diff
@@ -16,5 +16,11 @@
   - serviceaccounts
   verbs:
   - get
+- apiGroups:
+    - ""
+  resources:
+    - nodes/proxy
+  verbs:
+    - get
```

Comment:
Have you seen anything in the changelog regarding why it needs this added permissions?

Reply:
[This is the commit](https://github.com/aquasecurity/trivy-operator/commit/3911b5635070be32c34d60c1eafc248efff565b1#diff-6a6c7dca99aa76589f23c77ed3a9f61c830cd67d933e9887c321a015d373d174) related to [this issue](https://github.com/aquasecurity/trivy-operator/pull/1712) and more specifically a [CIS compliance check issue](https://github.com/aquasecurity/trivy-operator/issues/1687). Looks like i mistook chart version for apps version, will upgrade further.

Reply:
Looks like this is actually gated under `operator.accessGlobalSecretsAndServiceAccount`. Maybe we don't have that enabled or can disable it?

Reply:
Kind of looks like this new RBAC addition should not have been added under it though as it's regarding pulling images from private registries. Messy.